pokemongo_bot/cell_workers/evolve_all_worker.py

```diff
@@ -75,13 +77,12 @@ def _should_run(self):
                                .format(lucky_egg_count-1), 'green')
                 else:
                     logger.log('Failed to use lucky egg!', 'red')
-                    skip_evolves = True
-        elif use_lucky_egg: #lucky_egg_count is 0
-            # Skipping evolve so they aren't wasted
-            logger.log('No lucky eggs... skipping evolve!', 'yellow')
-            skip_evolves = True
+                    return False
 
-        return skip_evolves
+        else: # lucky_egg_count is 0
+            # Skipping evolve so pokemon aren't wasted
+            logger.log('No lucky eggs... skipping evolve!', 'yellow')
```

Comment:
OK. Isn't that what it is supposed to do? The idea is to skip evolving if you want to use lucky eggs but don't have any, otherwise the Pidgeys you stacked will be wasted.